### PR TITLE
tweaked slow search to filter edge cases

### PR DESCRIPTION
**Pull request by AaronWitter**
- added edge case tweak
- added tests

### CODE REVIEW

File: src/main/java/com/extractives/templates/ServersideSearchView.java [hunk 1]
```diff
@@ -24,8 +24,16 @@ public ServersideSearchView(String query, ExtractivesDao dao) {
     this.dao = dao;
   }
 
+  /**
+   * Checks query is not black or just whitespace, calls server search method using query.
+   * @return list of search elements.
+   */
```
Comment (KaneMorgan) [inline src/main/java/com/extractives/templates/ServersideSearchView.java]: :banana:
